### ISSUE TITLE
Document activities used by rucio-daemons 

### ISSUE THREAD

**Issue by dynamic-entropy** [rucio/documentation#326]
Will it be possible for an expert to Document the list of standard activities used by various daemons? 
Though you get some hints from the name, it is not entirely clear why certain transfers are happening. 

A few examples would be
- Data Consolidation
- Data Rebalancing
- Recovery

_Originally posted by @dynamic-entropy in https://github.com/rucio/rucio/discussions/6738_

**Comment by maxnoe** [comment]
Yes, please! A short description of the responsibilities of each daemon and what they are required for is missing from the docs.

**Comment by voetberg** [comment]
Closed by https://github.com/rucio/documentation/pull/346 - if there are additional details needed there, please make a new issue with the requested updates
